File: client/src/app/Models/Order.ts
import { Role } from './Role';

export class Order{
    order_ID:number;
    saleofDate:Date;
    total:number;
    customer_ID:number;
    employee_ID:number;
    customer_Name:any;
    employee_Name:any
    role!: Role;
    token?: string;
    /**
     *
     */
    constructor(order_ID:number,saleofDate:Date,
        total:number,
        customer_ID:number,
        employee_ID:number, customer:any,employee:any) {
            this.order_ID = order_ID;
           this.saleofDate =saleofDate;
           this.total =total;
           this.customer_ID =customer_ID;
           this.employee_ID =employee_ID;
           this.customer_Name = customer;
           this.employee_Name = employee;
    }
}

File: webapi/StoreManagement/Models/Order.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace StoreManagement.Models
{
    public class Order
    {
        [Key]
        public int Order_ID { get; set; }
        public int Employee_ID { get; set; }
        public int Customer_ID { get; set; }
        public DateTime SaleofDate { get; set; }
        public double Total { get; set; }
        public Customer Customer { get; set; }
        public Employee Employee { get; set; }
    }
}
